wrangler.toml

```diff
@@ -45,6 +45,14 @@ class_name = "SchedulerDO"
 name = "TOOLS_DO"
 class_name = "ToolsDO"
 
+[[durable_objects.bindings]]
+name = "IMAGES_DO"
+class_name = "ImageGeneratorDO"
+
+[[durable_objects.bindings]]
+name = "METADATA_DO"
+class_name = "TokenMetadataDO"
+
 # Durable Object migrations.
 # Docs: https://developers.cloudflare.com/workers/wrangler/configuration/#migrations
 [[migrations]]
```

Comment:
Also needs a migration, each time new durable objects get added a new block has to be added here. Can go below v2.

```
[[migrations]]
tag = "v3"
new_classes = ["ImageGeneratorDO", "TokenMetadataDO"]
```